Sum of digits of a number.py

```diff
@@ -1,7 +1,13 @@
 q=0
-n=int(input("Enter Number: "))
-while(n>0):
- r=n%10
- q=q+r
- n=n//10
-print("Sum of digits is: "+str(q))
+while True:
+    n=input("enter a number:")
+    if n.isnumeric():
+        n=int(n)
+        break
+    else:
```

Comment:
### Encapsulate all of this in funciton please.

Reply:
Okay.